Website/src/rendering/src/stories/Speakers/FeaturedSpeakers.stories.tsx

```diff
@@ -120,6 +120,26 @@ export const Default = Template.bind({});
 Default.args = {
   fields: {
     data: {
+      source: {
+        title: { value: 'Featured speakers' },
+        content: { value: 'lorem ipsum' },
+        numberOfSpeakers: { value: '6' },
+        callToActionLink: {
+          jsonValue: {
+            value: {
+              href: '/speakers',
+              text: 'View all speakers',
+              anchor: '',
+              linktype: 'internal',
+              class: '',
+              title: '',
+              target: '',
+              querystring: '',
+              id: '',
+            },
+          },
+        },
+      },
       item: {
         children: {
           results: [speaker, speaker1, speaker2, speaker3, speaker4],
```

Comment:
Please update the speakers objects to have:
- a role that is displayed
- a transparent background square picture
- Any other detail that would make the story to look as close as the expected component on the website.

Reply:
This was not done. I am unresolving the conversation